src/probabilistic.jl

```diff
@@ -28,7 +28,6 @@ function Probabilistic(
     if quantile_count != length(percentiles)
         throw(ArgumentError("The amount of elements in the data doesn't match the length of the percentiles"))
     end
-    data = handle_normalization_factor(input_data, normalization_factor)
```

Comment:
Is this not needed anymore?

Reply:
This logic is already invoked on line 26.